packages/salesforcedx-vscode-replay-debugger/test/unit/states/frameStateUtil.test.ts

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { expect } from 'chai';
+import 'mocha';
```

Comment:
You shouldn't need this. I was able to pull in your branch, removed this import, and compiled fine. Can you look into what's different about your setup? `git clean -xfd` may help.

Reply:
I have no idea why this was initially necessary. I just removed it locally and there's suddenly no problem but when I'd created this file, the mocha imports were all showing up as undefined.